src/mainnet/main-cudos-blast.md

```diff
@@ -0,0 +1,411 @@
+# Cudos Blast
+
+Cudos Blast is a Node.js CLI (command line interface) tool for working with the Cudos blockchain. You can scaffold, compile and test your **Rust** smart contracts. JavaScript and Rust testing is supported.
```

Comment:
it this the same documentation as the one in `src/`build`?

Reply:
Yes. The only differences are the ones related with the chain-id.